Files/DataModels/FilesystemItemsOperationDataModel.cs

```diff
@@ -62,7 +67,15 @@ public List<FilesystemOperationItemViewModel> ToItems(Action updatePrimaryButton
             // Add conflicting items first
             foreach (var item in ConflictingItems)
             {
-                items.Add(new FilesystemOperationItemViewModel(updatePrimaryButtonEnabled, optionGenerateNewName, optionReplaceExisting, optionSkip)
+                var (IconData, OverlayData, IsCustom) = await associatedInstance.FilesystemViewModel.LoadIconOverlayAsync(item.SourcePath, 64u);
+
+                BitmapImage icon = null;
+                await CoreApplication.MainView.DispatcherQueue.EnqueueAsync(async () =>
```

Comment:
When using `LoadIconOverlayAsync` the returned IconData can be null, better to add a check for this.

Reply:
Thank you!